### PR TITLE
Machines update

### PR DESCRIPTION
**Pull request by SonicHDC**
## About the PR

Updates autolathe and uniform printer.
And finally resprited Discount Dan's

**Media**

image
off
broken

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: SonicDC
- add: Added three more directions for autolathe!
- fix: Fixed incorrect rotation of the uniform printer!
- tweak: Discount Dan's sprites updated!

### CODE REVIEW

**Comment by Flareguy** [comment]
While these sprites are good, they're pretty hard to read as an autolathe. I'm not really sure if this is a good idea, but if you want to continue it, I would definitely add some extra stuff on the rotational sprites so that it's more recognizable.

**Comment by SonicHDC** [comment]
Well, I added a light bulb on top to show power. No more ideas yet.

**Comment by potato1234x** [comment]
isnt the discount dans meant to be poorly sprited?

File: Resources/Textures/Structures/Machines/VendingMachines/discount.rsi/meta.json [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
-    },
-    {
-      "name": "off"
-    }
-  ]
-}
+{
+  "version": 2,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Redesigned by SonicDC#9046",
```
Comment (deltanedas) [inline Resources/Textures/Structures/Machines/VendingMachines/discount.rsi/meta.json:4]: you still need to include vg copyright if modified

File: Resources/Textures/Structures/Machines/VendingMachines/discount.rsi/meta.json [hunk 2]
```diff
@@ -1,27 +1,29 @@
...
+      "name": "broken"
+    },
+    {
+      "name": "normal-unshaded",
+      "delays": [
+        [
+          0.4,
+          0.2,
+          0.4,
+		  0.4,
```
Comment (deltanedas) [inline Resources/Textures/Structures/Machines/VendingMachines/discount.rsi/meta.json:20]: sus indent

File: Resources/Textures/Structures/Machines/autolathe.rsi/meta.json [hunk 3]
```diff
@@ -1,51 +1,119 @@
...
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c6e3401f2e7e1e55c57060cdf956a98ef1fefc24",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "icon",
+	  "directions": 4
```
Comment (deltanedas) [inline Resources/Textures/Structures/Machines/autolathe.rsi/meta.json:12]: indent for every directions line is sus

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Textures/Structures/Machines/VendingMachines/discount.rsi/meta.json (hunk 2 above)
Comment (SonicHDC) [reply Resources/Textures/Structures/Machines/VendingMachines/discount.rsi/meta.json:20]: Whats sus?

**Pull request closed without merging**